lighthouse-core/lib/network-recorder.js

```diff
@@ -82,8 +82,14 @@ class NetworkRecorder {
   }
 
   onResourceChangedPriority(data) {
-    this.networkManager._dispatcher.resourceChangedPriority(data.requestId,
-      data.newPriority, data.timestamp);
+    // TODO: this.networkManager._dispatcher.resourceChangedPriority is set to
+    // undefined when onResourceChangedPriority is triggered in
+    // gatherers/offline.js. The underlying cause may need to be investigated.
+    // In the meantime, explicitly check that it's a function.
+    if (typeof this.networkManager._dispatcher.resourceChangedPriority === 'function') {
```

Comment:
This shouldn't be happening, so I suspect we're initializing something at the wrong time. We should hunt this down.